git-keeper-robot/gkeeprobot/keywords/ServerCheckKeywords.py

```diff
@@ -68,15 +68,23 @@ def submission_test_results_email_exists(self, username, course_name, assignment
             raise GkeepRobotException('No submission test result email for {}, {}, {}'.format(username, course_name,
                                                                                     assignment_name))
 
+    def submission_test_results_email_does_not_exist(self, username, course_name, assignment_name):
+        subject = 'submission test results'
+        result = control.run_vm_python_script('keeper', 'email_to.py',
+                                              username, subject)
+        if result != 'False':
+            raise GkeepRobotException('Submission test result email exists for {}, {}, {}'.format(username, course_name,
+                                                                                    assignment_name))
+
     def email_exists(self, to_user, subject_contains=None, body_contains=None):
         result = control.run_vm_python_script('keeper', 'email_to.py',
                                               to_user, subject_contains, body_contains)
         if result != 'True':
             raise GkeepRobotException('No email to {} containing subject {} and body {}'.format(to_user, subject_contains,
                                                                                       body_contains))
 
-    def email_does_not_exist(self, username):
-        result = control.run_vm_python_script('keeper', 'no_email_to.py',
+    def new_account_email_does_not_exist(self, username):
+        result = control.run_vm_python_script('keeper', 'no_account_email_to.py',
```

Comment:
Is there a reason for creating a separate VM script for this one rather than following the pattern of `submission_test_results_email_does_not_exist()`? Having some of the email keywords use their own scripts and others use `email_to.py` could get confusing.

Reply:
This is needed as a separate script.  If `no_email_to.py` returns `False`, we don't know if it is because the email didn't match or because no email existed.  The new function ensures that no email exists to the user.

Reply:
That makes sense, but since it is checking that there is no email at all to the user I think the original `no_email_to.py` filename is actually more descriptive, and the name `email_to.py` is the one that should be changed to `matching_email_exists.py` or something like that.

Reply:
New commit renames this as you suggested.